chain/client/src/chunk_inclusion_tracker.rs

```diff
@@ -102,4 +149,69 @@ impl ChunkInclusionTracker {
         }
         banned_chunk_producers.into_iter().collect_vec()
     }
+
+    fn get_chunk_info(&self, chunk_hash: &ChunkHash) -> Result<&ChunkInfo, Error> {
+        // It should never happen that we are missing the key in chunk_hash_to_chunk_info
+        self.chunk_hash_to_chunk_info
+            .get(chunk_hash)
+            .ok_or(Error::Other(format!("missing key {:?} in ChunkInclusionTracker", chunk_hash)))
+    }
+
+    pub fn chunk_header(&self, chunk_hash: &ChunkHash) -> Result<&ShardChunkHeader, Error> {
```

Comment:
nit: maybe `get_chunk_header`?

Also I would consider uniting it with `chunk_endorsements` and calling `get_endorsed_chunk_header`. Conceptually we need them together, for older protocol versions endorsements can be empty, and getting them separately makes me slightly uncomfortable - like the caller has to prepare chunk on its own, while it is actually ready.

Same applies to `get_prev_chunk_headers_and_chunk_endorsements` name and maybe even to `Block` API.

Reply:
Added `get_` prefix. I'll work on the get_prev_chunk_headers_and_chunk_endorsements and Block API in a future PR.